accounts/views.py

```diff
@@ -0,0 +1,27 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+from django.http.response import HttpResponseRedirect
+from django.views.generic import TemplateView, CreateView
+from django.contrib.auth import login
+from django.urls import reverse_lazy
+
+from .forms import SignUpForm
+
+
+class IndexView(TemplateView):
+  template_name = "accounts/index.html"
+
+class HomeView(TemplateView):
+  template_name = "accounts/home.html"
+
+class SignUpView(CreateView):
+  form_class = SignUpForm
+  template_name = "accounts/signup.html"
+  success_url = reverse_lazy("accounts:home")
+
```

Comment:
User = get_user_model()とした上で
model = Userを追加してください。
```suggestion
class SignUpView(CreateView):
  model = User
  form_class = SignUpForm
  template_name = "accounts/signup.html"
  success_url = reverse_lazy("accounts:home")

```